WORKSPACE

```diff
@@ -216,6 +216,16 @@ go_rules_dependencies()
 
 go_register_toolchains()
 
+http_archive(
```

Comment:
these probably need to go to https://github.com/bazelbuild/rules_docker/blob/master/repositories/repositories.bzl so that if you pull this repo as an external repo you can get these deps too

Reply:
As discussed offline, this is not necessary. Gazelle is only used to generate BUILD files when Go code in rules docker changes. As such the gazelle target in rules docker is of no use to repos that import rules docker.